obi_one/alb.tf

```diff
@@ -0,0 +1,40 @@
+resource "aws_lb_target_group" "obi_one_private_tg" {
+  #ts:skip=AC_AWS_0492
+  name        = "obi-one-private"
+  port        = 8000
+  protocol    = "HTTP"
+  target_type = "ip"
+  vpc_id      = var.vpc_id
+
+  lifecycle {
+    create_before_destroy = true
+  }
+
+  health_check {
+    enabled  = true
+    path     = "${var.root_path}/health"
+    protocol = "HTTP"
+  }
+}
+
+resource "aws_lb_listener_rule" "obi_one_private_listener_rule" {
+  listener_arn = var.private_alb_listener_arn
+  priority     = 710
+
+  action {
+    type             = "forward"
+    target_group_arn = aws_lb_target_group.obi_one_private_tg.arn
+  }
+
+  condition {
+    path_pattern {
+      values = ["${var.root_path}*"]
+    }
+  }
+
+  condition {
```

Comment:
If this service is meant to be accessible from 0.0.0.0/0 since inception, wouldn't make sense to delete all this `allowed_source_ip_cidr_blocks ` logic? 

I think it made sense when we were at EPFL/BBP but not sure it is useful anymore